docs/manual.txt

```diff
@@ -1908,13 +1908,23 @@ the following:
 
   This is also useful for DISQUS thread migration!
 
+* Allows you to export your comments with each post
+* Exports information on attachments per post
 * Will try to convert the content of your posts. This is *not* error free, because
```

Comment:
put info about your compiler here, drop this paragraph (you should really push the wordpress compiler)